frontend/express/public/localization/dashboard/dashboard.properties

```diff
@@ -155,6 +157,11 @@ taskmanager.last-7days = Last 7 days
 taskmanager.last-30days = Last 30 days
 taskmanager.manually-table-remind = These are the one-time and auto-refreshing reports manually created by you and your team
 taskmanager.automatically-table-remind = These are the reports automatically created when a query taks a long time to complete
+taskmanager.select-origin = Select origin
+taskmanager.auto = Auto 
+taskmanager.manual = Manual
```

Comment:
I think this is from merge conflict? now there are two taskmanager.auto and taskmanager.manual with different values and same for taskmanager.select-origin